crates/pulsar-core/src/pdk/process_tracker.rs

```diff
@@ -89,6 +96,20 @@ impl ProcessTrackerHandle {
         let r = self.tx.send(TrackerRequest::UpdateProcess(request));
         assert!(r.is_ok());
     }
+
+    /// Check if the process with the given PID is a descendant of the given image.
+    pub async fn is_descendant_of(&self, pid: Pid, image: String) -> Result<bool, TrackerError> {
```

Comment:
Since we've decided to convert errors to false below, there's no way this could possibly return `Err<TrackerError>`, so the return value should be `bool`